rkyv/Cargo.toml

```diff
@@ -52,6 +52,7 @@ validation = ["alloc", "bytecheck", "rend/validation"]
 bitvec_alloc = ["bitvec/alloc"]
 tinyvec_alloc = ["tinyvec/alloc"]
 uuid_std = ["uuid/std"]
+uuid_validation = ["uuid/std", "bytecheck/uuid"]
```

Comment:
Not happy introducing a new feature flag, esp. when it goes against the suggestion here:
https://github.com/uuid-rs/uuid/issues/515 but this does get a struct to implement `CheckBytes` when holding a `Uuid`. 

Happy to implement alternatives like rolling `bytecheck/uuid` into `validation`

Reply:
This should be possible without a new feature flag by adding `"bytecheck?/uuid"` to the `uuid` feature.